ir/node.cpp

```diff
@@ -18,11 +18,20 @@ limitations under the License.
 #include "ir/json_loader.h"
 
 #include "node.h"
+// #include <signal.h>
```

Comment:
Left a commented include here.

Reply:
yes, this is paired with the commented code a bit below too

Reply:
Ah I see, a comment about this could be useful.